dotnet-sdk-80/snapcraft.yaml

```diff
@@ -47,6 +47,54 @@ parts:
         craftctl set version="${DOTNET_SDK_VERSION}"
       fi
 
+  mounts:
+    plugin: dump
+    source: .
+    after:
+      - dotnet-sdk
+    override-build: |
+      RUNTIME_VERSION=$(ls ${CRAFT_STAGE}/usr/lib/dotnet/shared/Microsoft.NETCore.App/ | grep 8.0)
+      SDK_VERSION=$(ls ${CRAFT_STAGE}/usr/lib/dotnet/sdk | grep 8.0)
+      
+      if [ "${CRAFT_ARCH_BUILD_FOR}" = "amd64" ]; then
+        ARCH="x64"
+      elif [ "${CRAFT_ARCH_BUILD_FOR}" = "arm64" ]; then
+        ARCH="arm64"
+      else
+        echo "Unknown architecture (${CRAFT_ARCH_BUILD_FOR})"
+        exit 1
+      fi
+
+      for file in mounts/*; do
+        sed -i "s/{RT_VERSION}/$RUNTIME_VERSION/g" $file
```

Comment:
nitpick: The following is just my preference: I think renaming `RT_VERSION` to `RUNTIME_VERSION` would improve clarity

Reply:
Done!